### PR TITLE
typo - do instead of don't

### PR DESCRIPTION
**Pull request by catherinetsang**

### CODE REVIEW

**Comment by ashmaroli** [comment]
@catherinetsang Thank you for improving our docs.
Though the change is fine as is, I now realize that the paragraph itself doesn't flow smoothly while reading.

Will you be able to rephrase the whole paragraph to read better? If not, this will get merged anyways.

**Comment by catherinetsang** [comment]
@ashmaroli Sure thing. Let me try to rephrase it

File: docs/_docs/installation/windows.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ Bash on Ubuntu on Windows is still under development, so you may run into issues
 
 ## Encoding
 
-If you use UTF-8 encoding, make sure that no `BOM` header characters exist in your files. If they don't, Jekyll will break. This is especially relevant when you're running Jekyll on Windows.
```
Comment (catherinetsang) [inline docs/_docs/installation/windows.md:100]: I opted to shorten the paragraph to a single sentence by removing unnecessary words

Comment (ashmaroli) [reply docs/_docs/installation/windows.md:100]: Layman users will not know what `BOM` represents. So asking them to *remove `BOM`* wouldn't exactly be user-friendly.
How does one remove something they don't see nor understand..?

Comment (catherinetsang) [reply docs/_docs/installation/windows.md:100]: Thanks for the comment. I made a new commit

**Review by DirtyF — APPROVED** [review]
Thanks @catherinetsang 🙏🏻

**Comment by DirtyF** [comment]
@jekyll: merge +docs

**Pull request merged**